packages/block-editor/src/components/link-control/index.js

```diff
@@ -404,20 +404,24 @@ function LinkControl( {
 								createSuggestionButtonText
 							}
 							hideLabelFromVision={ ! showTextControl }
+							// Passing the Button component as a suffix prop
+							suffix={
+								showActions ? null : (
```

Comment:
We can also skip passing a prop when a suffix isn't needed, ie.

```suggestion
								showActions ? undefined : (
```

Reply:
Yes, I will update that.